bool closest_point_mode
---
bool success
